src/oncall/api/v0/populate.py

```diff
@@ -45,6 +46,6 @@ def on_post(req, resp, schedule_id):
     scheduler = load_scheduler(scheduler_name)
     schedule = get_schedules({'id': schedule_id})[0]
     check_team_auth(schedule['team'], req)
-    scheduler.populate(schedule, start_time, (connection, cursor))
+    scheduler.populate(schedule, start_time, (connection, cursor), req, resp, table_name)
```

Comment:
The temporary table details should be left out of the call to populate/preview, to allow other schedulers flexibility in defining their own populate/preview logic.